src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud/config.jelly

```diff
@@ -40,15 +40,15 @@
     </f:entry>
 
     <f:entry title="${%Connection Timeout (seconds)}" field="connectTimeout">
-        <f:textbox default="${descriptor.defaultConnectTimeout}"/>
+        <f:number min="${descriptor.defaultConnectTimeout}" default="${descriptor.defaultConnectTimeout}" checkMethod="post"/>
```

Comment:
Curious why you added `checkMethod="post"` here. It should be harmless (in fact it really ought to be the default in the Jenkins form lib), but normally it is needed only when the validation method might have side effects and/or might involve large values that cannot fit safely in a query string.

Reply:
I just picked up `@RequirePOST` from another validation method which was added through a security ticket, although I agree in this case it is pretty harmless.

Reply:
> another validation method which was added through a security ticket

Probably an SSRF.